### PR TITLE
use section and question description in correct places. fixes #38

### LINKED ISSUE #38: Question description used as section description, section description never used
### Expected behaviour

Question description appears under question title, section description appears under section title

### Actual behaviour

Question description appears under section title, section description never used

### Steps to reproduce the behaviour

1. create Q in author
2. Enter section title & description
3. Enter question title & description
4. Add any answer (fill in details)
5. Preview

### PR DESCRIPTION
**Pull request by WickyNilliams**
### What is the context of this PR?
Fixes issue where question description was being used as section description, and question description is never used.

### How to review 
Follow steps as described in issue, confirm descriptions appear in correct place

### CODE REVIEW

**Comment by WickyNilliams** [comment]
[Question description appearing in wrong place in runner (and section description never appearing anywhere)](https://trello.com/c/J77VkPNz/308-question-description-appearing-in-wrong-place-in-runner-and-section-description-never-appearing-anywhere)

File: src/eq_schema/Block.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
   QuestionPage: "Questionnaire",
   InterstitialPage: "Interstitial"
 };
 
 class Block {
-  constructor(title, page) {
+  constructor(title, description, page) {
     this.id = "block-" + page.id.toString();
     this.title = getInnerHTML(title);
-    this.description = getInnerHTML(convertPipes(page.description));
```
Comment (samiwel) [inline src/eq_schema/Block.js]: Wouldn't we still want to convert piped values for page titles?

Comment (WickyNilliams) [reply src/eq_schema/Block.js]: Yes, but that's a separate bug, i think. Not related to this change?

Comment (samiwel) [reply src/eq_schema/Block.js]: Okay, fair enough 👍

Comment (WickyNilliams) [reply src/eq_schema/Block.js]: Hmm actually need to investigate, seems piping isn't working at all now :-/

**Pull request merged**